packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -586,8 +586,11 @@ export default class NewVm extends BaseComponent {
   _getNetworkPredicate = createSelector(
     this._getIsInPool,
     this._getIsInResourceSet,
-    (isInPool, isInResourceSet) => network =>
-      isInResourceSet(network.id) || isInPool(network)
+    () => this.state.state.template,
+    (isInPool, isInResourceSet, template) => network =>
+      (isInResourceSet(network.id) || isInPool(network)) &&
```

Comment:
I think it should be `pool === undefined ? isInResourceSet(network.id) : isInPool(network)`.